Pollo/Extensions/UIColor+Shared.swift

```diff
@@ -8,44 +8,30 @@
 import UIKit
 
 extension UIColor {
-
+    
+    @nonobjc static let darkestGray = UIColor(white: 32.0 / 255.0, alpha: 1.0)
+    @nonobjc static let polloGreen = UIColor(red: 41.0 / 255.0, green: 192.0 / 255.0, blue: 158.0 / 255.0, alpha: 1.0)
+    @nonobjc static let blueyGrey = UIColor(red: 158.0 / 255.0, green: 167.0 / 255.0, blue: 179.0 / 255.0, alpha: 1.0)
```

Comment:
is `blueyGrey` the actual name of the color? sounds kind of funny 😂
maybe change to `blueGrey`